### PR TITLE
(#242) Trial Description general tests

### LINKED ISSUE #242: Trial Description page : general page components
### Acceptance Criteria

```gherkin
Scenario: User is able to see the trial description on desktop
    Given screen breakpoint is set to "desktop"
    Given user is navigating to "/about-cancer/treatment/clinical-trials/search/r?loc=0&rl=2"
The the page title is "Clinical Trials Search Results"
When user clicks on 1 trial result
    Then the page title is "Crizotinib in Treating Patients with Stage IB-IIIA Non-small Cell Lung Cancer That Has Been Removed by Surgery and ALK Fusion Mutations (An ALCHEMIST Treatment Trial)"
    And "< Back to search results" button as link is displayed
    And "This clinical trial matches:" appears below the title
    And search criteria accordion is displayed
    And "Start Over" link has a href "/about-cancer/treatment/clinical-trials/search/advanced"
    And trial status is "Status: active"
    And button "Open all" is displayed
    And button "Close all" is displayed
    And trial description accordion is displayed
    And button "Print" is displayed
    And button "Email" is displayed
    And the following delighters are displayed
      | delighter    | href                                                       | title                               | text                                                                  |
      | cts-livehelp | /contact                                                   | Have a question? We're here to help | Chat with us:  LiveHelp Call us: 1-800-4-CANCER(1-800-422-6237)       |
      | cts-which    | /about-cancer/treatment/clinical-trials/search/trial-guide | Which trials are right for you?     | Use the checklist in our guide to gather the information you’ll need. |
```
```gherkin
  Scenario Outline: User is able to see the trial description on mobile and tablet by directly navigating to trial
    Given screen breakpoint is set to "<breakpoint>"
    Given user is navigating to "/about-cancer/treatment/clinical-trials/search/v?id=NCI-2014-01507&loc=0&rl=2"
    Then the page title is "Crizotinib in Treating Patients with Stage IB-IIIA Non-small Cell Lung Cancer That Has Been Removed by Surgery and ALK Fusion Mutations (An ALCHEMIST Treatment Trial)"
    And "< Back to search results" button as link is not displayed
    And "This clinical trial matches:" appears below the title
    And search criteria accordion is displayed
    And "Start Over" link has a href "/about-cancer/treatment/clinical-trials/search/advanced"
    And trial status is "Status: active"
    And button "Open all" is displayed
    And button "Close all" is displayed
    And trial description accordion is displayed
    And button "Print" is not displayed
    And button "Email" is displayed
    And the following delighters are displayed
      | delighter    | href                                                       | title                               | text                                                                  |
      | cts-livehelp | /contact                                                   | Have a question? We're here to help | Chat with us:  LiveHelp Call us: 1-800-4-CANCER(1-800-422-6237)       |
      | cts-which    | /about-cancer/treatment/clinical-trials/search/trial-guide | Which trials are right for you?     | Use the checklist in our guide to gather the information you’ll need. |

    Examples:
      | breakpoint |
      | mobile     |
      | tablet     |
```
```gherkin
  Scenario: user can see search criteria specified for trial search
    Given user is navigating to "/about-cancer/treatment/clinical-trials/search/v?a=40&id=NCI-2014-01507&loc=0&rl=2"
    Then the page title is "Crizotinib in Treating Patients with Stage IB-IIIA Non-small Cell Lung Cancer That Has Been Removed by Surgery and ALK Fusion Mutations (An ALCHEMIST Treatment Trial)"
    And "This clinical trial matches:" appears below the title
    And search criteria accordion is displayed
    And button "Show Search Criteria"  is displayed
    When user clicks on "Show Search Criteria" button
Then button "Hide Search Criteria"  is displayed
    Then search criteria table is displayed with following
      | Category | Your Selection |
      | Age      | 40             |
```

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- *required dependency for identifying the problem, scope and then solution for this issue*
- [ ] Short Spike Description - Issue #9999

### Tasks
- [ ] Short Task Description - Issue #9999
- [ ] Short Task Description - Issue #9999

### Technical Notes
- More notes, or technical nuances/direction

### PR DESCRIPTION
**Pull request by kate-mashkina**
- added tests
Closes #242

### CODE REVIEW

**Review by seyilonge-nci — APPROVED** [review]

File: cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature [hunk 1]
```diff
@@ -0,0 +1,111 @@
+Feature:  As a user, I want to be able to view trial description page with all its components
```
Comment (bryanpizzillo) [inline cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature:1]: 1. We need to test to make sure that when we go directly to a trial description without being on search. This is used by the Listing App and clinical trials pretty URLS. This adds the following steps:
* `"< Back to search results" button as link is not displayed`
* `"This clinical trial matches: \"all trials\"" does not appear below the title`
* `search criteria table is not displayed` -- ok, this one already exists...

Comment (bryanpizzillo) [reply cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature:1]: 2. Given the nature of this page, we should be validating the metadata on the page. We are not checking:
* Title
* og:title
* og:description
* description
* Canonical URL
* og:image

Comment (bryanpizzillo) [reply cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature:1]: 3. The viewing page supports both the NCI ID and the NCT ID as valid IDs. See:
* https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/v?id=NCI-2014-01507
* https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/v?id=NCT02201992

(Both of these are the same trial)

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
Some additional test cases

File: cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature (hunk 1 above)
Comment (kate-mashkina) [reply cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature:1]: taken care of

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**